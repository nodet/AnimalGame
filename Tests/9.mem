cat
No
salmon
Does it live in water?
Yes
No
dolphin
Is it a mammal?
