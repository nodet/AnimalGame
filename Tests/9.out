Hi, let's play the animal game. You can quit at any time by typing "Quit."

Think of an animal.
Is your animal a cat? Oh no. What was it? What is a yes/no question to tell a salmon from a cat? Thanks! Let's play again.
Think of an animal.
Does it live in water? Is your animal a salmon? Oh no. What was it? What is a yes/no question to tell a dolphin from a salmon? Thanks! Let's play again.
Think of an animal.
Does it live in water? Is it a mammal? Is your animal a dolphin? Ha! I win! Let's play again
Think of an animal.
Does it live in water? Is your animal a cat? Ok. Bye!
